modules/test/playwright/tests/staging-configuration-web/main/stagingConfiguration.spec.ts

```diff
@@ -61,6 +72,79 @@ test('Check if local staging can be enabled', async ({
 	await stagingConfigurationPage.enableLocalStaging({});
 });
 
+test(
+	'Validate friendlyURL with special characters',
+	{tag: ['@LPS-89116']},
+	async ({
+		apiHelpers,
+		applicationsMenuPage,
+		journalPage,
+		page,
+		pagesAdminPage,
+		productMenuPage,
+		stagingConfigurationPage,
+	}) => {
+		const siteName: string = getRandomString();
+
+		await applicationsMenuPage.goToSites();
+
+		const site = await apiHelpers.headlessSite.createSite({
+			name: siteName,
+		});
+
+		apiHelpers.data.push({id: site.id, type: 'site'});
+
+		await journalPage.goto(site.friendlyUrlPath);
+		await productMenuPage.goToPages();
+
+		const pageName = getRandomString();
+		await pagesAdminPage.createNewPage({
+			name: pageName,
+		});
+
+		const basicWebContentStructureId =
+			await getBasicWebContentStructureId(apiHelpers);
+
+		const webContentName = 'Special Char aŐb';
+		await apiHelpers.jsonWebServicesJournal.addWebContent({
+			content: 'Web Content Content',
+			ddmStructureId: basicWebContentStructureId,
+			groupId: site.id,
+			titleMap: {en_US: webContentName},
+		});
+
+		await productMenuPage.openProductMenuIfClosed();
+		await productMenuPage.goToWebContent();
```

Comment:
Could we navigate directly via URL with journalPage.goto(site.friendlyUrlPath), or is using the product menu an intentional part of the test?

Reply:
True!